apptentive-feedback/src/main/java/apptentive/com/android/feedback/conversation/ConversationManager.kt

```diff
@@ -178,7 +180,12 @@ internal class ConversationManager(
         val existingConversation = loadExistingConversation()
         if (existingConversation != null) {
             Log.i(CONVERSATION, "Loaded an existing conversation")
```

Comment:
Android fetches configuration after manifest expiry and the value serialized will be used. However, I cannot serialize metrics value as part of the configuration as it would break the entire serialization. Hence, using the shared pref to use it in between the launches.